### DISCUSSION TITLE
Arewa 24

### DISCUSSION THREAD

**Discussion by shuaibuubah** [iptv-org/community#646] [Q&A]
***Channel Name:*** Arewa24
***Channel ID:** Arewa24.ng
***Country:*** Nigeria
***Language:*** Hausa
***Notes (optional):*** xxx

**Comment by BellezaEmporium** [comment]
Use iptv-org/iptv#1958 if you have channel requests concerning this country.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19649
